src/rebar_paths.erl

```diff
@@ -196,7 +198,9 @@ app_groups(Targets, State) ->
 get_paths(deps, State) ->
     rebar_state:code_paths(State, all_deps);
 get_paths(plugins, State) ->
-    rebar_state:code_paths(State, all_plugin_deps).
+    rebar_state:code_paths(State, all_plugin_deps);
+get_paths(runtime, State) ->
+    rebar_state:code_paths(State, all_deps).
```

Comment:
hm, the gotcha with that one is that unsetting runtime paths will also unset the deps that are _not_ in the runtime paths. I'm not sure if that's a problem in practice though.

Reply:
Mmm, I guess we could always get the runtime deps twice at most: the first time to either set or unset the paths and the second time to get the apps themselves :shrug:

Reply:
Yeah. At least if we don't call it often (i.e. just for xref) I'm not feeling too bad about the cost there. It shouldn't be super long if the recursive check function is made a bit generic.